src/libraries/System.Net.Security/src/Resources/Strings.resx

```diff
@@ -455,4 +455,7 @@
   <data name="net_ssl_renegotiate_data" xml:space="preserve">
     <value>Received data during renegotiation.</value>
   </data>
+  <data name="net_android_ssl_api_level_unsupported" xml:space="preserve">
```

Comment:
nit: maybe have `sni_hostname` somewhere in the resource name? The current name makes me think the message would be more generic than it is.

Reply:
Will work it in a follow up.